src/ui_fsmenu/singleplayersetupbox.cc

```diff
@@ -55,23 +55,23 @@ SinglePlayerActivePlayerGroup::SinglePlayerActivePlayerGroup(UI::Panel* const pa
                   settings,
                   id),
      tribe_(this,
-            (boost::format("dropdown_type%d") % static_cast<unsigned int>(id)).str(),
+            (boost::format("dropdown_type%d") % static_cast<unsigned>(id)).str(),
```

Comment:
```suggestion
            (boost::format("dropdown_tribe%d") % static_cast<unsigned>(id)).str(),
```

Reply:
done

Reply:
The dropdown *still* has the wrong internal name ;)

Reply:
upsi, sry, just looked at the static_cast.
fixed now :)